src/menus/icon-menu.jsx

```diff
@@ -67,6 +71,16 @@ const IconMenu = React.createClass({
     };
   },
 
+  componentClickAway(e) {
+    if (!this.state.open) {
+      return;
+    }
+
+    if (!(this.refs.popover && this.refs.popover.contains(e.target))) {
```

Comment:
Could we use this following opposite approach.
I think that it would be better to add a `onRequestClose` property
to the `Popover` component to make it "controllable" as we are doing with the `Dialog`
or the `LeftNav`.